README.md

```diff
@@ -221,6 +246,13 @@ won't help you, and you'll need to write a lower-level CMake
 file. This README can't cover all the possible situations here, but
 the `nowide` and `catch` CMake files are both solid examples.
 
+## How To Release
+
+1. Update [CHANGELOG.md](CHANGELOG.md) with release notes based on
+``git log `git describe --abbrev=0 --tags`..HEAD``
+1. Update the version in the project declaration of [CMakeLists.txt](CMakeLists.txt)
```

Comment:
"Note: You can specifically mark how far back your leatherman release is backwards compatible [here](https://github.com/puppetlabs/leatherman/blob/master/LeathermanConfigVersion.cmake.in#L4), for instance, if releasing leatherman `0.0.7` and your release is compatible with `0.0.6` leatherman (i.e. only new features), you could change `PACKAGE_VERSION` to `0.0.6`."
or something like that?